### PR TITLE
Fix strings hardcoded concatenation issues

### PR DESCRIPTION
**Pull request by pedro-mendonca**
Fixes issue #12

### CODE REVIEW

File: src/WPBT/WPBT_Core.php [hunk 1]
```diff
@@ -267,7 +267,7 @@ public function get_next_version( $preferred_version ) {
 			$next_version = __( 'next development version', 'wordpress-beta-tester' );
 		} else {
 			// show all versions that may come next.
-			$next_version = implode( ' or ', $next_version ) . ', ' . __( 'whichever is released first', 'wordpress-beta-tester' );
+			$next_version = wp_sprintf( __( 'version %l', 'wordpress-beta-tester' ), $next_version ) . ', ' . __( 'whichever is released first', 'wordpress-beta-tester' );
```
Comment (afragen) [inline src/WPBT/WPBT_Core.php]: I had no idea about this function, `wp_sprintf()`. Interesting.

Comment (pedro-mendonca) [reply src/WPBT/WPBT_Core.php]: It is indeed. I'm trying to change it from "and" to "or" :)

Comment (afragen) [reply src/WPBT/WPBT_Core.php]: We need to check the requirements of this function. We are limited in that we must currently support WP 3.1 and PHP 5.2. I think the use of `...$args` in the function might be an issue. Need to investigate.

Comment (afragen) [reply src/WPBT/WPBT_Core.php]: It still might be ok. Just need to make sure.

Comment (afragen) [reply src/WPBT/WPBT_Core.php]: Looks like the filter `wp_sprintf_l` is where the `and` comes from.

**Pull request merged**